src/ess/reflectometry/conversions.py

```diff
@@ -61,3 +61,28 @@ def specular_reflection() -> dict:
     graph["theta"] = theta
     graph["Q"] = reflectometry_q
     return graph
+
+
+def tof_to_wavelength(data_array: sc.DataArray,
+                      wavelength_edges: sc.Variable,
+                      graph: dict = specular_reflection()) -> sc.DataArray:
```

Comment:
```suggestion
                      graph: dict = None) -> sc.DataArray:
```
And later
```.py
graph = graph if graph is not None else specular_reflection()
```
in order to avoid a mutable default argument.